include/Core/Request/LogListAction.php

```diff
@@ -65,9 +65,20 @@ public function view_log_message() {
 
 					</table>
 
-					<?php
-					echo wpautop( $log_item['message'] );
+					<div id="tabs">
+						<ul>
+							<li><a href="#tabs-1">HTML</a></li>
```

Comment:
Oops, this didn't occur to me. I'll make a checklist before submitting PR. Fixed it now.